bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/RPV_GluinoGluinoTo3jets/RPV_GluinoGluinoToJets_M-1000_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set loop_optimized_output True
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```

Comment:
please follow these suggestions https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
I am not sure where to set these options from https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

for MG5_aMC 2017 requests use 

$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS = reweight_PDF

Reply:
in the run_card :-)